app/assets/javascripts/discourse/app/components/sidebar/edit-navigation-menu/categories-modal.gjs

```diff
@@ -174,4 +185,93 @@ export default class extends Component {
         this.saving = false;
       });
   }
+
+  <template>
+    <EditNavigationMenuModal
+      @title="sidebar.categories_form_modal.title"
+      @disableSaveButton={{this.saving}}
+      @save={{this.save}}
+      @showResetDefaultsButton={{gt
+        this.siteSettings.default_navigation_menu_categories.length
+        0
+      }}
+      @resetToDefaults={{this.resetToDefaults}}
+      @deselectAll={{this.deselectAll}}
+      @deselectAllText={{i18n "sidebar.categories_form_modal.subtitle.text"}}
+      @inputFilterPlaceholder={{i18n
+        "sidebar.categories_form_modal.filter_placeholder"
+      }}
+      @onFilterInput={{this.onFilterInput}}
+      @resetFilter={{this.resetFilter}}
+      @filterSelected={{this.filterSelected}}
+      @filterUnselected={{this.filterUnselected}}
+      @closeModal={{@closeModal}}
+      class="sidebar__edit-navigation-menu__categories-modal"
+    >
+      <form class="sidebar-categories-form">
+        {{#if (gt this.filteredCategoriesGroupings.length 0)}}
+          {{#each this.filteredCategoriesGroupings as |categories|}}
+            <div
+              class="sidebar-categories-form__row"
+              style={{borderColor (get categories "0.color") "left"}}
```

Comment:
I know it's not your doing here either, but I wonder if since we have category css variables, can we remove the inline styles and use css with a selector with category ID included?